Vagrant/provision/setup.sh

```diff
@@ -1,24 +1,33 @@
-#!/bin/sh
+#!/bin/bash
 
 # update
 pacman -Syyu --noconfirm
 
+# check if jdk and jre exists, if not download
+if ! pacman -Qs jdk-openjdk > /dev/null && ! pacman -Qs jre-openjdk; then
```

Comment:
What will happen if **jdk-openjdk** is installed but **jre-openjdk** is not?

Reply:
I see. It will skip and `jre-openjdk` would never install. I have to change `&&` to `||`.